includes/Modules/Analytics_4/Report/Metric_Filter/Between_Filter.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics_4\Report\Metric_Filter\In_List_Filter
+ *
+ * @package   Google\Site_Kit\Modules\Analytics_4\Report\Metric_Filter
+ * @copyright 2023 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Analytics_4\Report\Metric_Filter;
+
+use Google\Site_Kit_Dependencies\Google\Service\AnalyticsData\Filter as Google_Service_AnalyticsData_Filter;
+use Google\Site_Kit_Dependencies\Google\Service\AnalyticsData\FilterExpression as Google_Service_AnalyticsData_FilterExpression;
+use Google\Site_Kit_Dependencies\Google\Service\AnalyticsData\BetweenFilter as Google_Service_AnalyticsData_BetweenFilter;
+use Google\Site_Kit_Dependencies\Google\Service\AnalyticsData\NumericValue as NumericValue;
+
+/**
+ * Class for parsing the metric between filter.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Between_Filter implements Between_Filter_Interface {
```

Comment:
Unless I am missing something, I don't think the use of an explicit interface here is really needed... We don't use one for the existing dimension filter utility classes, and it doesn't seem needed for anything...

I'd suggest removing the `Between_Filter_Interface` and `Numeric_Filter_Interface` interfaces unless there's a reason we do need them.